release/data_processing_tests/workloads/streaming_shuffle.py

```diff
@@ -0,0 +1,177 @@
+import time
+import json
+import ray
+import numpy as np
+from typing import List
+from tqdm import tqdm
+
+from ray.cluster_utils import Cluster
+
+num_nodes = 4
+num_cpus = 4
+partition_size = int(500e6)  # 500MB
+# Number of map & reduce tasks == num_partitions.
+# Number of objects == num_partitions ^ 2.
+num_partitions = 200
+# There are two int64 per row, so we divide by 8 * 2 bytes.
+rows_per_partition = partition_size // (8 * 2)
+object_store_size = 20 * 1024 * 1024 * 1024  # 20G
+
+system_config = {
```

Comment:
Won't these be set automatically once object spilling is turned on?

Reply:
Yes. But we'd like to run this in the current release, and object spilling will be turned off for the current release. I can create other PR to fix it later.